en-US/Design.xml

```diff
@@ -215,9 +215,9 @@
 		 </section>
 		</section>
 		<section id="figures-illustrations">
-			<title>Figures, Illustrations, and Screen Captures</title>
+			<title>Figures, Illustrations, and Screenshots</title>
 			<para>
-				Refer to the <citetitle>Figures</citetitle> section in the <citetitle>IBM Style Guide</citetitle> for general advice about using figures, illustrations, and screen captures.
+				Refer to the <citetitle>Figures</citetitle> section in the <citetitle>IBM Style Guide</citetitle> for general advice about using figures, illustrations, and screenshots.
```

Comment:
There are many references to IBM Style throughout the guide. What do you think in general about whether (and where) to provide direct links, bearing in mind that Red Hatters have access and others do not? Currently, section 7.1 has a link and access instructions for Red Hat. Is that sufficient? 

Also, I noticed that section 7.1 currently refers to CMOS 17. CMOS 18 is now published; should we refer to that version instead?

Reply:
Maybe we can put a comment about links to IBM somewhere more prominent, somewhere in the front matter, and then just use them as needed.

And yes, I'd always refer to the latest version of CMOS, so maybe say that instead of using ver numbers to save maintenance.